src/containers/Dashboard.vue

```diff
@@ -301,11 +298,14 @@ export default Vue.extend({
     /**
      * Creates a course on frontend with either user or API data
      */
-    createCourse(course: FirestoreSemesterCourse, isRequirementsCourse: boolean): AppCourse {
+    createAppCourseFromCornellRosterCourse(
```

Comment:
Would this be the function that would be made global so that we can remove references to `$parent`?

Reply:
Yes, this is my intention.